workorders.lic

```diff
@@ -77,6 +80,57 @@ class WorkOrders
     end
   end
 
+
+  def carve_items(info, item, quantity)
```

Comment:
A good candidate for some refactoring, either here in `workorders` or `common-crafting`.

Don't worry about it as part of this PR, but we'll want to look into it later.